pizza_app/css/style.css

```diff
@@ -0,0 +1,251 @@
+
+.page-footer .copyright {
+    padding-top: 10px;
+    font-size: 12px;
+}
+
+/* BUTTONS */
+
+.btn {
+    color: #1f2230;
+    background: #f2b632;
+    border: 1px solid transparent;
+    border-radius: 5px;
+    font-size: 14px;
+    font-weight: bold; 
+    text-transform: uppercase;
+    box-shadow: -1px 1px 8px rgba(0, 0, 0, 0.6);
+    cursor: pointer;
+}
+
+.btn:hover,
+.btn:focus {
+    background: #f5cb5c;
+    box-shadow: -2px 2px 16px rgba(0, 0, 0, 0.8);
+    outline: none;
+}
+
+.btn:active {
+    background: #e8dcb9;
+    box-shadow: -4px 4px 24px rgba(0, 0, 0, 0.8);
+    outline: none;
+}
+
+.btn-wrapper {
+    display: flex;
+}
+
+.page-header .btn {
+    width: 40px;
+    height: 30px;
+}
+
+.page-header .btn-add img {
+    width: 20px;
+}
+
+.page-header .btn-login {
+    margin: 0 10px;
+}
+
+.page-header .btn-login span {
+    display: none;
+}
+
+.page-main .btn-add {
+    width: 300px;
+    margin: 20px auto 30px auto;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 10px;
+}
+
+.page-main .btn-add img {
+    width: 35px;
+    margin-right: 10px;
+}
+
+/* QUEUE */
+
+.queue {
+    max-width: 1300px;
+    margin: 0 auto;
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+    grid-gap: 30px;
+    justify-items: center;
+}
+
+.order {
+    width: 300px;
+    display: flex;
+    flex-direction: column;
+    padding: 20px; 
+    background: #1f2230;
+    border: 2px solid #f2b632;
+    border-radius: 5px;
+    box-shadow: -1px 1px 8px rgba(0, 0, 0, 0.6);
+}
+
+.order img {
+    width: 260px;
+}
+
+.order .details {
+    display: grid;
+    grid-template: repeat(2, 1fr) / repeat(2, 1fr);
+    grid-gap: 15px;
+    margin-top: 20px;
+    padding: 0 10px;
+}
+
+.order-number,
+.order-price {
+    text-align: right;
+}
+
+/* ADAPTIVE */
+
+@media (min-width: 481px) {
+    .page-header,
+    .page-footer {
+        padding-right: 10px;
+        padding-left: 10px;
+    }
+    
+    .page-header .current-time .fa-clock {
+        display: initial;
+    }
+
+    .page-header .title img {
+        margin-right: 10px;
+    }
+
+    .page-header .title h1 {
+        font-size: 24px;
+    }
+}
+
+@media (min-width: 768px) {
+    .page-header,
+    .page-footer {
+        padding-right: 20px;
+        padding-left: 20px;
+    }
+
+    .page-footer {
+        flex-direction: row;
+        padding: 20px;
+    }
+
+    .page-footer .copyright {
+        padding-top: 0;
+    }
+}
+
+@media (min-width: 1025px) {
+    .page-header .current-time {
+        width: 200px;
+        font-size: 18px;
+    }
+
+    .page-header .title img {
+        width: 80px;
+        height: 80px;
+        margin: 5px 10px 5px 0;
+    }
+
+    .page-header .title h1 {
+        font-size: 48px;
+    }
+
+    .page-header .btn-login {
+        width: 150px;
+    }
+
+    .page-header .btn-login span {
+        display: initial;
+    }
+
+    .page-main {
+        margin-top: 110px;
+    }
+}
```

Comment:
Add new line to your code. __Always__ It makes `git diff` look much better.

You can setup your editor to ensure it on save.

Reply:
fixed it, thanks